### PR TITLE
added fix for '/system' not in /proc/mounts

### PR DESCRIPTION
**Pull request by abhaynayar**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

### CODE REVIEW

File: Document/0x05b-Basic-Security_Testing.md [hunk 1]
```diff
@@ -1644,6 +1644,10 @@ $ cd /system/xbin
 $ chmod 755 tcpdump
 ```
 
+In certain production builds, you might encounter an error `mount: '/system' not in /proc/mounts`.
```
Comment (cpholguera) [inline Document/0x05b-Basic-Security_Testing.md]: Hi @abhaynayar and thanks for sending the PR :)
do you have some more details on this? any specific phone or Android version?

Comment (abhaynayar) [reply Document/0x05b-Basic-Security_Testing.md]: This worked on my Moto g5s plus running Android 10.

Comment (cpholguera) [reply Document/0x05b-Basic-Security_Testing.md]: Do you think this is an issue since Android 10?

Comment (cpholguera) [reply Document/0x05b-Basic-Security_Testing.md]: Please provide any references you may have. that'd be very helpful :)

Comment (abhaynayar) [reply Document/0x05b-Basic-Security_Testing.md]: I don't know whether this is in Android 10 issue or not (I think it is more to do with certain custom ROMs), not able to test due to lack of Android 10 devices. As for reference I only have a stackoverflow [link](https://stackoverflow.com/questions/28009716/how-can-i-remount-my-android-system-as-read-write-in-a-bash-script-using-adb).

Comment (sushi2k) [reply Document/0x05b-Basic-Security_Testing.md]: Thanks for sharing. So the stack overflow links is 5 years old :-) Meaning it's a problem with certain hardware devices it seems, not the android version.

File: Document/0x05b-Basic-Security_Testing.md [hunk 2]
```diff
@@ -1644,6 +1644,10 @@ $ cd /system/xbin
...
+
+In that case, you can replace the above line `$ mount -o rw,remount /system;` with `$ mount -o rw,remount /`.
```
Comment (sushi2k) [inline Document/0x05b-Basic-Security_Testing.md]:
```suggestion
In that case, you can replace the above line `$ mount -o rw,remount /system;` with `$ mount -o rw,remount /`, as described in [this Stackoverflow post](https://stackoverflow.com/a/28018008 "How can I remount my Android/system as read-write in a bash script using adb?").
```

**Comment by abhaynayar** [comment]
@sushi2k Yeah that makes sense, committed.

File: Document/0x05b-Basic-Security_Testing.md [hunk 3]
```diff
@@ -1646,7 +1646,7 @@ $ chmod 755 tcpdump
 
 In certain production builds, you might encounter an error `mount: '/system' not in /proc/mounts`.
 
-In that case, you can replace the above line `$ mount -o rw,remount /system;` with `$ mount -o rw,remount /`.
+In that case, you can replace the above line `$ mount -o rw,remount /system;` with `$ mount -o rw,remount /`, as described in [this Stackoverflow post](https://stackoverflow.com/a/28018008 "How can I remount my Android/system as read-write in a bash script using adb?").
```
Comment (cpholguera) [inline Document/0x05b-Basic-Security_Testing.md]: minor naming correction
```suggestion
In that case, you can replace the above line `$ mount -o rw,remount /system;` with `$ mount -o rw,remount /`, as described in [this Stack Overflow post](https://stackoverflow.com/a/28018008 "How can I remount my Android/system as read-write in a bash script using adb?").
```

**Comment by abhaynayar** [comment]
👍

**Pull request merged**